hibernate-core/src/main/java/org/hibernate/boot/registry/internal/StandardServiceRegistryImpl.java

```diff
@@ -94,6 +95,24 @@ private void applyServiceRegistrations(List<StandardServiceInitiator<?>> service
 		}
 	}
 
+	/**
+	 * Not intended for general use. We need the ability to stop and "reactivate" a registry to allow
+	 * experimentation with technologies such as GraalVM, Quarkus and Cri-O.
+	 */
+	public synchronized void resetAndReactivate(BootstrapServiceRegistry bootstrapServiceRegistry,
+												List<StandardServiceInitiator<?>> serviceInitiators,
+												List<ProvidedService<?>> providedServices,
```

Comment:
should we use generics wildcard to allow collection of child class instances to be able to pass in? Like:
```
public synchronized void resetAndReactivate(BootstrapServiceRegistry bootstrapServiceRegistry,
												List<? extends StandardServiceInitiator<?>> serviceInitiators,
												List<? extends ProvidedService<?>> providedServices,
```